test/fixtures/scenarios/single-hana-single-app/e77ab524-c4e4-5381-bb59-5af8639d52a4_host_discovery.json

```diff
@@ -19,6 +19,18 @@
       "fe80::8d3:33ff:feee:9701",
       "fe80::ec17:89ff:feb6:a456"
     ],
+    "netmasks": [
+      "8",
+      "128",
+      "16",
+      "24",
+      "24",
+      "64",
+      "24",
+      "24",
+      "64",
+      "64"
+    ],
```

Comment:
We have 11 ip_addresses but only 10 netmasks, i guess we need one more :)

Reply:
damn, really good catch

Reply:
Fixed!